CMakeLists.txt

```diff
@@ -755,6 +755,17 @@ if(UMF_FORMAT_CODE_STYLE)
     endif()
 endif()
 
+find_package(Python3)
+if(Python3_FOUND)
```

Comment:
Please, check also the minimum required version.

Reply:
done.